cve_bin_tool/cvedb.py

```diff
@@ -483,6 +493,24 @@ def init_database(self) -> None:
 
         self.db_close()
 
+    def populate_purl2cpe(self):
+        """Transfers data from PURL2CPE database to CVE database."""
+
+        purl2cpe_conn = sqlite3.connect(self.cachedir / "purl2cpe/purl2cpe.db")
+        purl2cpe_cursor = purl2cpe_conn.cursor()
+
+        cve_conn = sqlite3.connect(self.dbpath)
+        cve_cursor = cve_conn.cursor()
+
+        purl2cpe_cursor.execute("SELECT purl, cpe FROM purl2cpe")
+        cve_cursor.execute("DELETE from purl2cpe")
```

Comment:
This led me down a rabbit hole of the question "is DELETE or DROP TABLE faster in SQLite?"

Sounds like they should be comparable now because they've optimized DELETE with no arguments, so this code is fine as is, but they *both* used to be very slow compared to other databases: https://sqlite.org/speed.html

So I learned a thing today!  If we do have slowdowns with this it does sound like the recommended alternative is to go back to having it in a separate db and delete the file (which is much faster).  Given that this is something that will run once a day I'm not too worried about adding a bit of time, but it's interesting and we should keep an eye on our cache jobs to see if it makes a big impact or not.